### PR TITLE
Fix: EditPriceAndBeneficiaryModal

### LINKED ISSUE #2248: [BUG][Creation] 'EditPriceAndBeneficiaryModal' resets the beneficiary/price fields
## 🎉 Description

When configuring the price, if the checkbox for I'm the beneficiary, the price is resetted. The same happens when the the Make it free checkbox is turned on (it resets the benefitiary)

* [x] 🐛 This is a bug report.
* [ ] 📈 This is a feature request.

### LINKED ISSUE #2249: [Creation][Enhancement] 'EditPriceAndBeneficiaryModal'
## 🎉 Description

Enhance the 'EditPriceAndBeneficiaryModal' 

* [ ] Adjust the margins of the elements in the modal
    * Reduce the margin between the input and the checkbox to `4px`
    * Increase the margin between fields price and beneficiary to `32px`

### PR DESCRIPTION
**Pull request by cyaiox**
This PR fixes the fields resetting values when toggling the checkboxes and updates some styles.

Closes #2248
Closes #2249

### CODE REVIEW

File: src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx [hunk 1]
```diff
@@ -332,7 +332,7 @@ export default class CreateSingleItemModal extends React.PureComponent<Props, St
       const view = isEmotesFeatureFlagOn && type === ItemType.EMOTE ? CreateItemView.THUMBNAIL : CreateItemView.DETAILS
       const data = await getItemData({ wearablePreviewController: previewController, type, model, contents, category })
       this.setState({ metrics: data.info, thumbnail: data.image, isLoading: false }, () => {
-        this.setState({ view })
+        this.setState({ view, fromView: CreateItemView.IMPORT })
```
Comment (cyaiox) [inline src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx:335]: Not related to this PR but fixes the back button for emotes workflow

File: src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx [hunk 2]
```diff
@@ -837,14 +837,14 @@ export default class CreateSingleItemModal extends React.PureComponent<Props, St
...
-    const { isLoading, contents } = this.state
+    const { isLoading, contents, fromView } = this.state
 
     return (
       <EditThumbnailStep
         isLoading={!!isLoading}
         blob={contents ? toWearableWithBlobs({ contents, isEmote: true }) : undefined}
         title={this.renderModalTitle()}
-        onBack={() => this.setState({ view: CreateItemView.DETAILS })}
+        onBack={() => this.setState({ view: fromView! })}
```
Comment (cyaiox) [inline src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx:847]: Not related to this PR but fixes the back button for emotes workflow

File: src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.types.ts [hunk 3]
```diff
@@ -45,7 +45,7 @@ export type StateData = {
   previewController?: IPreviewController
   weareblePreviewUpdated: boolean
 }
-export type State = { view: CreateItemView; itemSortedContents?: Record<string, Blob> } & Partial<StateData>
+export type State = { view: CreateItemView; fromView?: CreateItemView; itemSortedContents?: Record<string, Blob> } & Partial<StateData>
```
Comment (cyaiox) [inline src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.types.ts:48]: Not related to this PR but fixes the back button for emotes workflow

**Review by nicosantangelo — APPROVED** [review]
LGTM

**Review by juanmahidalgo — APPROVED** [review]
LGTM!

**Pull request merged**